docs/interactions/structured-messages/webview-sdk.md

```diff
@@ -0,0 +1,101 @@
+# About Webview SDK
+
+This SDK is meant to be used on pages linked to by messages/items with a "webview" target.
+
+This kind of target is useful to create specific interactions that are not provided by channels.
+
+The Webview SDK provides a generic way to interact with supported channels that can open your page in a webview which allows you to use the same target on different channels without having to write multiple integrations.
+
+## Supported channels
+
+The SDK supports the following channels:
+
+- Engage Messaging (Web only for now)
```

Comment:
Oh, I was rather thinking about a section where the webviews are supported. So in docs/interactions/web-messaging/webview.md 

Because for the webview SDK, it's kinda already said in the `## Supported features by channels` section no?

Reply:
The problem I have with the file you're suggesting is that it's specific to Engage Messaging, so maybe it should be moved in `docs/interactions/structured-messages` folder, WDYT?

Reply:
Ah yes indeed, good idea let's move it as you say.

FTR, an alternative idea could be to create a new "Webview" menu at the same level of the Structured message one. With overview being this page, and a webview SDK too. Probably better when we'll have more menus.

Reply:
Should I add a list of sources or structured_contents in general supporting webview in this page now that it has moved ?

Reply:
Yes please.

Reply:
By the way, do we still need this `The SDK supports the following channels:` part now if @aymebou adds it to the webview page?

Reply:
ok if you want.

Reply:
I added the section and the link in 7433c10